core/src/client/mod.rs

```diff
@@ -38,14 +38,21 @@ use futures_channel::{mpsc, oneshot};
 use futures_util::future::FutureExt;
 use futures_util::sink::SinkExt;
 use futures_util::stream::{Stream, StreamExt};
-use jsonrpsee_types::{Id, ParamsSer, SubscriptionId};
+use jsonrpsee_types::{Id, SubscriptionId};
 use serde::de::DeserializeOwned;
 use serde_json::Value as JsonValue;
+use crate::params::BatchRequestBuilder;
+use crate::traits::ToRpcParams;
 
+// Re-exports for the `rpc_params` macro.
 #[doc(hidden)]
 pub mod __reexports {
-	pub use crate::to_json_value;
-	pub use jsonrpsee_types::ParamsSer;
+	// Needs to be in scope for `ArrayParams` to implement it.
+	pub use crate::traits::ToRpcParams;
+	// Main builder object for constructing the rpc parameters.
+	pub use crate::params::ArrayParamsBuilder;
+	// Empty rpc parameters for empty macro.
+	pub use crate::params::EmptyParams;
```

Comment:
Ah ok, these are reexported for the macro. If this is still needed, I guess we'll be able to remove EmptyParams from this list when we update the macro :)